src/App.js

```diff
@@ -1,7 +1,10 @@
+import './scss/App.scss';
+import { Calculator } from './components/Calculator';
+
 function App() {
   return (
     <>
-      <h1>Hello World!</h1>
+      <Calculator />
     </>
   );
 }
```

Comment:
- [ ] Good job so far 👍 According to the requirements all your components should be class-based. Kindly convert this `app` component from a function to a class 😄